### PR TITLE
Add new test cases for RAddCommandParser

### LINKED ISSUE #353: Add new test cases for RAddCommandParser
Let's increase the code coverage to 100%.

### PR DESCRIPTION
**Pull request by Hikoya**
Resolves #353

### CODE REVIEW

File: src/test/java/foodwhere/logic/parser/RAddCommandParserTest.java [hunk 1]
```diff
@@ -77,6 +90,32 @@ public void parse_stallIndexFieldMissing_failure() {
                 expectedMessage);
     }
 
+    @Test
+    public void parse_stallIndexInvalid_failure() {
+        String expectedMessage = MESSAGE_INVALID_INDEX_ERROR;
+
+        // "hubby*" for index
```
Comment (clarence-chew) [inline src/test/java/foodwhere/logic/parser/RAddCommandParserTest.java:97]: This comment is useful, nice work!

File: src/test/java/foodwhere/model/common/NameTest.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package foodwhere.model.stall;
+package foodwhere.model.common;
```
Comment (clarence-chew) [inline src/test/java/foodwhere/model/common/NameTest.java:1]: Wondering why this is model.common instead of model.commons

**Review by clarence-chew — APPROVED** [review]
LGTM, though minor concern about package naming

File: src/test/java/foodwhere/model/common/NameTest.java (hunk 2 above)
Comment (Hikoya) [reply src/test/java/foodwhere/model/common/NameTest.java]: Fixed in latest commit.

File: src/test/java/foodwhere/logic/parser/RAddCommandParserTest.java (hunk 1 above)
Comment (Hikoya) [reply src/test/java/foodwhere/logic/parser/RAddCommandParserTest.java:97]: Thank you!

**Review by clarence-chew — APPROVED** [review]
LGTM.

**Pull request merged**